### ISSUE TITLE
Update the nested CMOR tables for their updates

### ISSUE THREAD

**Issue by treerink** [EC-Earth/ece2cmor3#804]
Update the nested CMOR tables for their updates. This time the update is more complicated. It seems that the PCMDI `cmip6-cmor-tables` repository ([see their issue 407](https://github.com/PCMDI/cmip6-cmor-tables/issues/407)) renamed their default master branch to 'main' in October 2023.

**Comment by treerink** [comment]
In 1b85499 the CMOR tables have been updated based on the HEAD of main (switched from master to main) following the renaming of their default branch. Although the latest update of this nested repository was almost one year ago I only saw (non EC-Earth related) changes in one file: `CMIP6_CV_UPDATE`.  So the update is rather limited.

**Comment by treerink** [comment]
Probably since PCMDI renamed the `master` branch to `main` branch, the state of our nested master branch got in a detached state. This is now solved with the latest update in 1b85499.

It is strongly recommended to run the script:
```
cd ${HOME}/cmorize/ece2cmor3/ece2cmor3/scripts
./revert-nested-cmor-table-branch.sh
```
If you are, for whatever reasons, in a deviated state before updating for this issue, you always can come back to the ece2cmor3 prior state for the nested CMOR tables by:
```
 cd ${HOME}/cmorize/ece2cmor3/ece2cmor3/resources/tables/
 gitc 4d4cbc4
```
